docs/resources/modelarts_resource_pool.md

```diff
@@ -24,62 +26,205 @@ resource "huaweicloud_modelarts_resource_pool" "test" {
 }
 ```
 
+### create a privileged resource pool
+
+```hcl
+variable "vpc_id" {}
+variable "subnet_id" {}
+variable "cce_cluster_id" {}
+variable "login_user_password" {}
+variable "security_group_ids" {
+  type = list(string)
+} 
+
+resource "huaweicloud_modelarts_resource_pool" "test" {
+  name        = "demo"
+  description = "This is a demo"
+  vpc_id      = var.vpc_id
+  subnet_id   = var.subnet_id
+  password    = var.login_user_password
+
+  clusters {
+    provider_id = var.cce_cluster_id
+  }
+  
+  resources {
+    flavor_id          = "modelarts.vm.cpu.8ud"
+    count              = 1
+    node_pool          = "test-name"
+    vpc_id             = var.vpc_id
+    subnet_id          = var.subnet_id
+    security_group_ids = var.security_group_ids
+
+    labels = {
+      "aaa" = "111"
+      "bbb" = "222"
+    }
+
+    tags = {
+      key   = "terraform"
+      owner = "value"
+    }
+
+    taints {
+      key    = "key"
+      value  = "value"
+      effect = "NoSchedule"
+    }
+  }
+
+  charging_mode = "prePaid"
+  period_unit   = "month"
+  period        = 1
+  auto_renew    = false
+}
+```
+
 ## Argument Reference
 
 The following arguments are supported:
 
 * `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource.
   If omitted, the provider-level region will be used. Changing this parameter will create a new resource.
 
-* `name` - (Required, String, ForceNew) The name of the dedicated resource pool.  
-    The name can contain `4` to `32` characters, only lowercase letters, digits and hyphens (-) are allowed.
-    The name must start with a lowercase letter and end with a lowercase letter or digit.
+* `name` - (Required, String, ForceNew) Specifies the name of the dedicated resource pool.  
+  The name can contain `4` to `32` characters, only lowercase letters, digits and hyphens (-) are allowed.
+  The name must start with a lowercase letter and end with a lowercase letter or digit.
 
   Changing this parameter will create a new resource.
 
-* `scope` - (Required, List) List of job types supported by the resource pool.  
-  The options are as follows:
-    + **Train**: training job.
-    + **Infer**: inference job.
-    + **Notebook**: Notebook job.
+* `scope` - (Optional, List) Specifies the list of job types supported by the resource pool. It can not be specified if
```

Comment:
how about put `scope` and `network_id` together?

also, we can writing like this: `This parameter is mandatory when network_id is specified.`

Reply:
done